gbpservice/nfp/orchestrator/config_drivers/heat_driver.py

```diff
@@ -445,6 +486,70 @@ def _get_all_heat_resource_keys(self, template_resource_dict,
                 resource_keys.append(key)
         return resource_keys
 
+    def _create_firewall_template(self, auth_token,
+                                  service_details, stack_template):
```

Comment:
Name should be update_firewall_template, since firewall template already exists, and we are updating existing template